__tests__/app.test.js

```diff
@@ -202,8 +210,50 @@ describe("app", () => {
         .post("/api/articles/100/comments")
         .send(inputBody)
         .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("article id not found");
+        .then(({ body: err }) => {
+          expect(err.msg).toBe("article id not found");
+        });
+    });
+  });
+  describe("PATCH /api/articles/:article_id", () => {
+    it("Request body accepts an object in the form { inc_votes: newVote }, updating votes, responds with the updated article", () => {
+      const inputBody = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/3")
+        .send(inputBody)
+        .expect(202)
+        .then(({ body: article }) => {
+          expect(article).toEqual({
+            article_id: 3,
+            title: "Eight pug gifs that remind me of mitch",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "some gifs",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 1,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+    });
+    it("same as above but with decrementing number of votes", () => {
```

Comment:
aim to keep each description self-contained, so it doesn't rely on any other to be understood